images/Dockerfile.kairos

```diff
@@ -15,6 +15,7 @@ RUN rm -rf /etc/ssh/ssh_host_*
 COPY which-init.sh /usr/local/bin/which-init.sh
 
 RUN sed -i -n '/KAIROS_/!p' /etc/os-release
+RUN if [ -f "/etc/kairos-release" ]; then sed -i -n '/KAIROS_/!p' /etc/kairos-release; fi
```

Comment:
We should be able to remove line 17 from now on, no?

Reply:
No idea, I wanted to be trough. IIRC this was added in case we run it twice or similar, so could it happen that we dump them into os-release and then run it again thus leaving stuff behind?

Reply:
I don't think so, but it doesn't hurt keeping it indeed

Reply:
Thinking about this, yeah I think you are rigth, both this and the one below cna be dropped as this affects only NEW built images